bin/hdfcoinc/pycbc_multiifo_coinc_statmap

```diff
@@ -151,6 +151,7 @@ f = fw(args.output_file)
 f.attrs['num_of_ifos'] = all_trigs.attrs['num_of_ifos']
 f.attrs['pivot'] = all_trigs.attrs['pivot']
 f.attrs['fixed'] = all_trigs.attrs['fixed']
+f.attrs['ifos'] = all_trigs.attrs['ifos']
```

Comment:
this needs to have the same `if 'ifos' in all_trigs.attrs` check as above

Reply:
check that the `ifos` attribute is staying similar throughout (does it always have spaces or not?)

Reply:
It is given with spaces in the coinc jobs. This takes a multiifo coinc job for the given detector combination, it will not run into error within a workflow.

Reply:
Actually to be precise the 'ifos' attribute is always given by ' '.join(sorted(args.ifos)) to be exactly reproducible.

Reply:
> this needs to have the same `if 'ifos' in all_trigs.attrs` check as above

Just a reminder for this - I think that as the two comments were for the same line, this got added to the same conversation as the other question on github

Reply:
I put in the check but it is unnecessary, this code should only get a coinc file from a pycbc_multiifo_coinc_findtrig job, which will always have an 'ifos' attribute. I think both checks should never return false.